dev/ci/ci-wrapper.sh

```diff
@@ -14,7 +17,16 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "${DIR}/../.." || exit 1
 
 export TIMED=1
-bash "${DIR}/${CI_SCRIPT}" 2>&1 | tee "$CI_NAME.log"
+if { [ -t 1 ] || [ "$CI" ]; } && command -v script > /dev/null; then
+  # prevent piping from disabling auto colors / enable auto colors in CI
+  if ! [ "$TERM" = dumb ]; then export TERM=fake; fi
```

Comment:
the printenv above doesn't have TERM, but `set -x` says it evaluates to `dumb` here
WTF

Reply:
apparently bash sets `TERM` to `dumb` as a normal variable (not exported) if it's unset.